pkg/kdump/kdump-view.jsx

```diff
@@ -41,14 +41,69 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "rhel-system-roles";
+    if (os_release.NAME.includes('Fedora')) {
+        role_name = "linux-system-roles";
+    }
```

Comment:
Is this correct for CentOS Stream? It would feel more correct the other way around: special-case RHEL (and possibly CentOS), and default ot linux-s-r on all other OSes, as that's the upstream name.

Reply:
It is, seems CentOS has https://repology.org/project/rhel-system-roles/versions

Note that only: Fedora, CentOS are supported, well and I guess rocky? etc. https://linux-system-roles.github.io/

One issue is that we have SUSE support for Kdump which uses a different format so that shouldn't show this automation script tab I suppose?